content/definition.md

```diff
@@ -5,8 +5,8 @@ template = "definition"
 
 Software is considered Ethical Source if it meets the following criteria:
 
-1. It is distributed with source code and can be freely used or combined with other software.
-1. It is developed or maintained in public view and welcoming of public contributions.
+1. It can be freely used or combined with other software.
+1. It is developed or maintained in public view and welcoming of public contributions, which necessarily requires public access to all of the software's source code.
```

Comment:
"It is available, developed, and maintained in public view and welcoming of public contributions."?

Reply:
Sure, that's more succint

Reply:
"The source code is available, developed, and maintained"?

I think you have to have the word "source code" in there somewhere to be clear.

Reply:
Good call, go ahead